config/initializers/session_store.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: "_velum_session"
+Rails.application.config.session_store :cookie_store,
+  key:    "_velum_session",
+  secure: Rails.env.production?
```

Comment:
i didn't know for sure that the failure is related to adding this header, at least the test error doesn't make sense in that regard, as all other tests are passing